lib/service/cfg.go

```diff
@@ -204,6 +205,9 @@ type Config struct {
 
 	// PluginRegistry allows adding enterprise logic to Teleport services
 	PluginRegistry plugin.Registry
+
+	// AuthServerFeatures contain flags for supported and unsupported features.
+	AuthServerFeatures proto.Features
```

Comment:
Nit: I think AuthServerFeatures is a bit too specific. How about ClusterFeatures? If you decide to rename, let's rename all authServerFeatures below as well.

Reply:
renamed